tests/k8s-azure/manifest/cluster-api/cluster-template-prow-ci-version-oot-credential-provider.yaml

```diff
@@ -214,7 +214,7 @@ spec:
     spec:
       identity: UserAssigned
       userAssignedIdentities:
-      - providerID: ${AZURE_USER_ASSIGNED_IDENTITY_PROVIDER_ID}
+      - providerID: /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/capz-ci/providers/Microsoft.ManagedIdentity/userAssignedIdentities/cloud-provider-user-identity
```

Comment:
We don't have a long-lived managed identity in the new sub with this specific name and resource group like we did in the old sub, so this should still use the variables CI_RG and USER_IDENTITY like the CAPZ templates:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/a498485f56254673096d2184b5db1ec950240693/templates/test/ci/cluster-template-prow.yaml#L142